c2corg_ui/static/js/documentediting.js

```diff
@@ -551,12 +586,28 @@ app.DocumentEditingController.prototype.formatOuting_ = function(outing) {
     // init empty conditions_levels for ng-repeat
     outing['locales'][0]['conditions_levels'] = [{'level_snow_height_soft': '', 'level_snow_height_total': '', 'level_comment': '', 'level_place': ''}];
   }
-
   return outing;
 }
 
 
 /**
+ * get parameters from the URL separated by &
+ * @returns {Object} url parameters
+ * @private
+ */
+app.DocumentEditingController.prototype.parseUrl_ = function() {
```

Comment:
Do you really need this?

When creating a route from a WP page the URL would be http://.../routes/add?waypoint=123456
When creating an outing from a route page the URL would be http://.../outings/add?route=123456

I also think we should consider extending the documentEditing directive depending on the document type. See https://github.com/c2corg/v6_ui/issues/246